corehq/apps/data_dictionary/util.py

```diff
@@ -229,3 +229,13 @@ def fields_to_validate(domain, case_type_name):
     }
     props = CaseProperty.objects.filter(**filter_kwargs)
     return {prop.name: prop for prop in props}
+
+
+@quickcache(['domain', 'case_type'], timeout=24 * 60 * 60)
+def get_smart_types_by_prop(domain, case_type):
```

Comment:
Here you are only filtering by `gps`, maybe rename this function to match what it is returning?

Reply:
I'm anticipating this being used for a couple other types in the medium-term future, but I guess it's not hard to change and there's always the likelihood that YAGNI.